src/next/components/InfoCols/InfoCols.tsx

```diff
@@ -0,0 +1,77 @@
+import { BiRightArrowAlt } from "react-icons/bi"
+import { InfoColsProps } from "~/common"
+import { SUPPORTED_ICONS_MAP, SupportedIconName } from "~/common/Icons"
+
+const InfoColsHeader = ({
+  title,
+  subtitle,
+}: Pick<InfoColsProps, "title" | "subtitle">) => (
+  <div className="w-full flex flex-col gap-7 items-start text-left">
+    <h1 className="text-[#333333] text-4xl font-normal leading-tight">
+      {title}
+    </h1>
+    {subtitle && <p className="text-xl font-normal">{subtitle}</p>}
+  </div>
+)
+
+const InfoBoxIcon = ({ icon }: { icon?: SupportedIconName }) => {
+  if (!icon) return null
+  const Icon = SUPPORTED_ICONS_MAP[icon]
+  return (
+    <div>
+      <Icon className="w-10 h-auto text-secondary" />
+    </div>
+  )
+}
+
+const InfoBoxes = ({ infoBoxes }: Pick<InfoColsProps, "infoBoxes">) => {
+  return (
+    <div className="grid grid-cols-1 gap-x-8 gap-y-12 md:grid-cols-2 xl:grid-cols-3">
+      {infoBoxes.map((infoBox, idx) => (
+        <div key={idx} className="flex flex-col gap-4 items-start text-left">
+          <InfoBoxIcon icon={infoBox.icon} />
+          <div className="flex flex-col gap-1 items-start text-[#2C2E34]">
```

Comment:
why is the color hardcoded here?

Reply:
originally hardcoded it as wasn't sure if this was just a special universal "black" for next theme or was intended to be site primary or secondary
will check with design and add the colours to config as required!

Reply:
checked with design and added
[b920866](https://github.com/isomerpages/isomer-components/pull/10/commits/b920866d3535d3306898c5e2794bcd7d6a33a0b5)